.github/workflows/rust_checks.yml

```diff
@@ -0,0 +1,125 @@
+name: Rust Static checks
+
+on:
+  pull_request:
+    paths:
+      - .github/workflows/rust_checks.yml
+      - Cargo.lock
+      - Cargo.toml
+      - charts/linkerd-control-plane/templates/destination-rbac.yaml
+      - charts/linkerd-crds/templates/policy/**
```

Comment:
Since this doesn't run integration tests, we don't need to run this on chart changes.

```suggestion
```